chat-microservice/app/tests/cassandra/test_chat_model.py

```diff
@@ -0,0 +1,62 @@
+import uuid
+
+from app.models.chat import Chat
+
+def test_create_a_single_record(cassandra_session):
+    model_instance = Chat.create(
+        chat_id = str(uuid.uuid4()),
+        users_id = [str(uuid.uuid4()), str(uuid.uuid4())],
+        users_name = ["Frank", "Pepe"]
+    )
+    assert Chat.objects().count() == 1
+
+def test_chat_id_must_be_unique(cassandra_session):
+    chat_id_unique = str(uuid.uuid4())
+    for _ in range(3):
+        Chat.create(
+            chat_id = chat_id_unique,
+            users_id = [str(uuid.uuid4()), str(uuid.uuid4())],
+            users_name = ["Frank", "Pepe"]
+        )
+    assert Chat.objects().count() == 1
+
+def test_find_all_franks_chat(cassandra_session):
+    frank_id = str(uuid.uuid4())
+    for _ in range(3):
+        Chat.create(
+            chat_id = str(uuid.uuid4()),
+            users_id = [frank_id, str(uuid.uuid4())],
+            users_name = ["Frank", "Pepe"]
+        )
+    for _ in range(3):
+        Chat.create(
+            chat_id = str(uuid.uuid4()),
+            users_id = [str(uuid.uuid4()), str(uuid.uuid4())],
+            users_name = ["Frank", "Pepe"]
+        )
+    assert Chat.objects().count() == 6
+    franks_chat = Chat.objects().filter(users_id__contains=frank_id).allow_filtering()
```

Comment:
In some places you treat user ids as strings, in others as uuids. If you treat them as str this test is flakey because of the `__contains` in the filter. UUIDs have guaranteed lengths, so if you will be using them it is impossible for one to contain another. If in the other hand, the `"1"` and `"2"` style user ids used in other tests and in the API docs is the one you would actually use, it would be flakey because an id `"12341294172"` can contain an id `"2"`

Reply:
This is gold, But I am too lazy right now to address it XD